data/oftenlinks.json

```
@@ -73,6 +73,16 @@
   "description": "Speiseplan Studierendenwerk Mensa Berliner Tor",
   "keywords": [],
   "url": "https://www.studierendenwerk-hamburg.de/speiseplan?l=170"
+}, {
+  "name": "FSR I2E Nextcloud - Onlinespeicher",
+  "description": "Nextcloud Instanz des FSR I2E",
+  "keywords": ["cloud", "nextcloud", "fsr"],
+  "url": "https://nc.i2e.fsr.haw-hamburg.de/"
+}, {
+  "name": "FSR I2E",
+  "description": "Gemeinschaftwebseite der des Fachschaftsrat Informatik und des Fachschaftsrat Informations- und Elektrotechnik",
+  "keywords": ["website", "fsr"],
+  "url": "https://i2e.fsr.haw-hamburg.de/"
```

I think the general website is more important than the specific nextcloud?

```suggestion
  "name": "FSR I2E",
  "description": "Gemeinschaftwebseite der des Fachschaftsrat Informatik und des Fachschaftsrat Informations- und Elektrotechnik",
  "keywords": ["website", "fsr"],
  "url": "https://i2e.fsr.haw-hamburg.de/"
}, {
  "name": "FSR I2E Nextcloud - Onlinespeicher",
  "description": "Nextcloud Instanz des FSR I2E",
  "keywords": ["cloud", "nextcloud", "fsr"],
  "url": "https://nc.i2e.fsr.haw-hamburg.de/"
```